bcipy/tasks/main_frame.py

```diff
@@ -228,15 +238,39 @@ def decide_state_update(self):
         self.list_epoch[-1]['decision'] = decision
         return decision
 
-    def prepare_stimuli(self):
+    # TODO: think about where timing, color, is_txt, num_sti should go.
+    # e.g. is_txt is predefined with the alphabet directly
+    def _prepare_stimuli(self, timing=[1, 0.2], color=['red', 'white'],
```

Comment:
Now that prepare_stimuli is private to the class, we should remove it from the class documentation.

Reply:
I removed documentation. 

Stimuli is generated for the main_frame and it is not required to be called outside. Therefore, I switched it to a private class. However, I am not %100 sure about my decision.

Reply:
I think it should go outside. It's easier to test in isolation.